src/hooks/screen/useChatInfoHook.ts

```diff
@@ -76,9 +80,9 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
 
   const handleShowArrow = (item: Member) => {
     if (params?.from === SIGNED) {
-      return item?.user_id !== myUserId?.myProfile?.user_id;
+      return item?.user?.userId !== signedProfileId;
     }
-    if (!isContainUrl(item?.user?.image) || item?.user?.name === ANONYMOUS_USER) {
+    if (!isContainUrl(item?.user?.profilePicture) || item?.user?.anon_user_info_emoji_name) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a change in the logic of `handleShowArrow` function. The new code checks if the user's profile picture is not a URL or if the user has an anonymous emoji name, then it returns false. This might cause issues if the user does not have a profile picture but has a valid non-anonymous name. Consider revising this logic to handle such cases.
```diff
-    if (!isContainUrl(item?.user?.profilePicture) || item?.user?.anon_user_info_emoji_name) {
+    if (!isContainUrl(item?.user?.profilePicture) && !item?.user?.anon_user_info_emoji_name) {
       return false;
     }
```
This change ensures that the function only returns false when both conditions are met: the user does not have a profile picture and does not have an anonymous emoji name.


<!-- This is an auto-generated comment by OSS CodeRabbit -->